lnd/fullnode/lnd.conf

```diff
@@ -0,0 +1,84 @@
+[Application Options]
+
+; The alias your node will use, which can be up to 32 UTF-8 characters in
+; length.
+alias=LNCM Default
+
+; The color of the node in hex format, used to customize node appearance in
+; intelligence services.
+color=#000000
+
+; The maximum number of incoming pending channels permitted per peer.
+maxpendingchannels=3
+
+; Adding an external IP will advertise your node to the network. This signals
+; that your node is available to accept incoming channels. If you don't wish to
+; advertise your node, this value doesn't need to be set. Unless specified
+; (with host:port notation), the default port (9735) will be added to the
+; address.
+;externalip=host:port
+
+; Instead of explicitly stating your external IP address, you can also enable
+; UPnP or NAT-PMP support on the daemon. Both techniques will be tried and
+; require proper hardware support. In order to detect this hardware support,
+; `lnd` uses a dependency that retrieves the router's gateway address by using
+; different built-in binaries in each platform. Therefore, it is possible that
+; we are unable to detect the hardware and `lnd` will exit with an error
+; indicating this. This option will automatically retrieve your external IP
+; address, even after it has changed in the case of dynamic IPs, and advertise
+; it to the network using the ports the daemon is listening on. This does not
+; support devices behind multiple NATs.
+; nat=false
+
+; Specify the interfaces to listen on for p2p connections.  One listen
+; address per line.
+; All ipv4 on port 9735:
+listen=0.0.0.0:9735
+
+; Specify the interfaces to listen on for gRPC connections.  One listen
+; address per line.
+rpclisten=0.0.0.0:10009
+
+; Specify the interfaces to listen on for REST connections.  One listen
+; address per line.
+; All ipv4 interfaces on port 8080:
+restlisten=0.0.0.0:8080
+
+; Debug logging level.
+; Valid levels are {trace, debug, info, warn, error, critical}
+; You may also specify <subsystem>=<level>,<subsystem2>=<level>,... to set
+; log level for individual subsystems.  Use btcd --debuglevel=show to list
+; available subsystems.
+; debuglevel=info
+debuglevel=info
+
+; Number of logfiles that the log rotation should keep. Setting it to 0 disables deletion of old log files.
+maxlogfiles=3
+
+; Max log file size in MB before it is rotated.
+maxlogfilesize=10
+
+; DANGER ZONE! Makes it impossible to restore wallets!
+; noseedbackup=1
+
+; Control what is allowed for TLS domain names
+; Definitely allow localhost
+; It is advisable to add your domain in case externalip maps to one
+;tlsextradomain=
+tlsextraip=127.0.0.1
+
+[Routing]
+; Save bandwidth by only downloading minimum channel filters necessary
+routing.assumechanvalid=1
```

Comment:
Isn't that the thing that caused the most recent lnd CVE?

I don't know.  Just asking now.